tests/lib/email/test_email_api.py

```diff
@@ -0,0 +1,246 @@
+import unittest
+from unittest import mock
+from unittest.mock import ANY, MagicMock
+
+from email_api.email_api import EmailApi
+from nose.tools import raises
+
+
+class EmailApiTests(unittest.TestCase):
+    TEST_ACCOUNTS = [
+        {
+            "name": "default",
+            "username": "test",
+            "password": "password",
+            "server": "test.server",
+            "port": 123,
+            "secure": False,
+            "smtp_auth": False,
+        },
+        {
+            "name": "test",
+            "username": "test",
+            "password": "password",
+            "server": "test.server",
+            "port": 123,
+            "secure": True,
+            "smtp_auth": True,
+        },
+    ]
+
+    def setUp(self) -> None:
+        self.api = EmailApi()
+
+    @raises(FileNotFoundError)
+    def test_load_template_file_not_found(self):
+        """
+        Tests that load_template can raise a FileNotFoundError
+        """
+        self.assertRaises(FileNotFoundError, self.api.load_template("test"))
```

Comment:
I think this is duplicated, this should just be `self.api.load_template("test")` since we already have the `@raises` decorator?